CHANGES.rst

```diff
@@ -1,5 +1,9 @@
 Changes
 -------
+1.3.3 (2021-08-20)
```

Comment:
nit: 1.3.4 ?

Reply:
🤦🏼 looks like test-version wasn't run for some reason to catch this, trying rename

Reply:
hah, looks like files with dashes aren't tested

Reply:
now @ 1.4.0, marking resolved